cypress/e2e/signIn.cy.js

```diff
@@ -12,23 +12,50 @@ describe('Sign In page', () => {
 
   before(() => {
     cy.task('db:clear');
-    cy.task('generateUser').then((generateUser) => {
-      user = generateUser;
+
+    // Створити нового користувача через API
+    const username = 'riot';
+    const email = 'riot@qa.team';
+    const password = '12345Qwert!';
+    // Використання команди для реєстрації користувача
+    cy.register(email, username, password).then((response) => {
```

Comment:
Verify that the `cy.register` command is correctly implemented and registers a user via the API. This is crucial for setting up the test environment with a valid user.